Month1/Week2/Section2/SpotifyProject/Playlist.cpp

```diff
@@ -21,7 +21,7 @@ void Playlist::setPlaylistTitle(std::string pPlaylistTitle)
 	playlistTitle = pPlaylistTitle;
 }
 
-std::string Playlist::getPlaylistTitle()
+const std::string Playlist::getPlaylistTitle()
```

Comment:
const before the function doesn't make the function const. It is actually returning a const value, which is actually not good practice hehe. To make a method const, the const keyword should be placed _after_ the method. So:
``` std::string Playlist::getPlaylistTitle() const```